package.json

```diff
@@ -56,12 +56,12 @@
     "browserama": "^3.2.0",
     "core-js": "^3.6.5",
     "debounce-promise": "^3.1.2",
-    "exponential-backoff": "^3.1.0",
+    "exponential-backoff": "^3.1.1",
     "genesys-cloud-client-logger": "^4.2.7",
     "limiter": "^1.1.0",
     "lodash.throttle": "^4.1.1",
     "lru-cache": "^6.0.0",
-    "stanza": "^12.17.2",
```

Comment:
Unless I'm missing something, I think we can bump to stanza 12.20.0. That includes the `queueMicrotask` fix.